docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -16,110 +16,59 @@ ms.assetid: fab89cb8-dac1-4e21-9d34-5eadd5213c05
 
 # Create an Azure service principal with Azure CLI 2.0
 
-If you plan to manage your app or service with Azure CLI 2.0,
-you should run it under an Azure Active Directory (AAD) service principal rather than your own credentials.
-This topic steps you through creating a security principal with Azure CLI 2.0.
+If you want to manage permissions for your app or service, you can do so through creating a service principal under Azure Active Directory. A service principal will allow you to set per-app permissions restrictions. This topic will run you through the steps for creating a service principal and setting its permissions.
```

Comment:
Service principals are about two things-running code w/ a different identity than a login, and running code with restricted permissions. So maybe use "control" or "restrict permissions for your app or service". We'll talk about the need to use a different idenity somewhere else in this or another topic.